src/librustc/session/config.rs

```diff
@@ -1194,8 +1194,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "prints the llvm optimization passes being run"),
     ast_json: bool = (false, parse_bool, [UNTRACKED],
         "print the AST as JSON and halt"),
-    query_threads: Option<usize> = (None, parse_opt_uint, [UNTRACKED],
-        "execute queries on a thread pool with N threads"),
+    threads: Option<usize> = (None, parse_opt_uint, [UNTRACKED],
```

Comment:
Is there any support for jobserver or is this a static quantity? Perhaps the documentation should be adjusted either way?

Reply:
It will use a jobserver once https://github.com/rust-lang/rust/pull/56946 lands